main.js

```diff
@@ -0,0 +1,252 @@
+      [7, 8, 9]
+    ];
+  }
+  reset() {
+    this.board = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ];
+    counter = 0;
+  }
+
+
+  checkWinner(gameBoard) {
+    var position1 = game.board[0][0];
+    var position2 = game.board[0][1];
+    var position3 = game.board[0][2];
+    var position4 = game.board[1][0];
+    var position5 = game.board[1][1];
+    var position6 = game.board[1][2];
+    var position7 = game.board[2][0];
+    var position8 = game.board[2][1];
+    var position9 = game.board[2][2];
+    if (position1 == "X" && position2 == "X" && position3 == "X") {
+      console.log("X wins top across");
+      game.reset();
+    } else if (position1 == "O" && position2 == "O" && position3 == "O") {
+      console.log("O wins top across");
+      game.reset();
+    } else if (position4 == "X" && position5 == "X" && position6 == "X") {
+      console.log("X wins middle across");
+      game.reset();
+    } else if (position4 == "O" && position5 == "O" && position6 == "O") {
+      console.log("O wins middle across");
+      game.reset();
+    } else if (position7 == "X" && position8 == "X" && position9 == "X") {
+      console.log("X wins bottom across");
+      game.reset();
+    } else if (position7 == "O" && position8 == "O" && position9 == "O") {
+      console.log("O wins bottom across");
+      game.reset();
+    } else if (position1 == "X" && position4 == "X" && position7 == "X") {
+      console.log("X wins left-side down");
+      game.reset();
+    } else if (position1 == "O" && position4 == "O" && position7 == "O") {
+      console.log("O wins left-side down");
+      game.reset();
+    } else if (position2 == "X" && position5 == "X" && position8 == "X") {
+      console.log("X wins middle down");
+      game.reset();
+    } else if (position2 == "O" && position5 == "O" && position8 == "O") {
+      console.log("O wins middle down");
+      game.reset();
+    } else if (position3 == "X" && position6 == "X" && position9 == "X") {
+      console.log("X wins right-side down");
+      game.reset();
+    } else if (position3 == "O" && position6 == "O" && position9 == "O") {
+      console.log("O wins right-side down");
+      game.reset();
+    } else if (position1 == "X" && position5 == "X" && position9 == "X") {
+      console.log("X wins diagonal");
+      game.reset();
+    } else if (position1 == "O" && position5 == "O" && position9 == "O") {
+      console.log("O wins diagonal");
+      game.reset();
+    } else if (position7 == "X" && position5 == "X" && position3 == "X") {
+      console.log("X wins diagonal");
+      game.reset();
+    } else if (position7 == "O" && position5 == "O" && position3 == "O") {
+      console.log("O wins diagonal");
+      game.reset();
+    } else if (counter >= 9) {
+      console.log("TIE!");
+      game.reset();
+    }
+  }
+};
+class Player {
+  constructor(symbol) {
+    this.symbol = symbol;
+    // NOT WORKING
+    // this.array = [];
+  }
+
+  play1() {
```

Comment:
Again, at the risk of being harsh, this is a really bad idea. You're repeating yourself a ton here and leaving all kinds of room for bugs and errors.